jerr/jerr.go

```diff
@@ -26,6 +27,15 @@ func (e JError) Print() {
 	fmt.Println(e.Error())
 }
 
+// Message to be presented to the user.
+func (e *JError) SetDisplayMessage(message string) {
```

Comment:
What happens when this gets called from multiple places?

Reply:
Yeah. Could add an `AppendDisplayMessage()`. Just wanted to delineate from the system trace that gets logged. It's TMI.